inbm/Changelog.md

```diff
@@ -14,14 +14,15 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
 ### Added
 - RTC 508495 Support INBC Config Append command
 - RTC 508497 Support INBC Config Remove command
+- RTC 500237 - Remove DBS messages from appearing in INBC
 
 ## 3.0.10 - 2022-03-28
 
 ## NEXT - 3.0.9 - 2022-03-08
```

Comment:
is this section for 3.0.10 or 3.0.9?

Reply:
@gblewis1 - good catch.  I missed that the numbers were messed up.  I think I fixed it.  Check again.